parlai/core/params.py

```diff
@@ -673,12 +673,20 @@ def add_parlai_args(self, args=None):
             type=int,
             help='batch size for minibatch training schemes',
         )
+        parlai.add_argument(
+            '-nw',
+            '--numworkers',
```

Comment:
nit: hyphen between num and workers?

Reply:
Or just —workers?

Reply:
i can switch to `--num-workers`. This was originally a flag from the [deprecated] PytorchDataTeacher, as it's passed into the pytorch dataloader `numworkers` field. Given the deprecation, we can change